plugins/woocommerce-blocks/assets/js/blocks/product-filter/inner-blocks/price-filter/frontend.ts

```diff
@@ -70,17 +70,28 @@ store< PriceFilterStore >( 'woocommerce/product-filter-price', {
 				type === 'min'
 					? Math.min(
 							Number.isNaN( value ) ? minRange : value,
-							maxRange - 1
+							maxPrice
 					  )
 					: minPrice;
 			const currentMaxPrice =
 				type === 'max'
 					? Math.max(
 							Number.isNaN( value ) ? maxRange : value,
-							minRange + 1
+							minPrice
 					  )
 					: maxPrice;
 
+			// In some occasions the input element is updated with the incorrect value.
```

Comment:
In my case, when the min and max price are equal, if I continue to move the max price to lower than the min price, the prices in the text inputs are correct, but the thumb doesn't return to the correct position.